### PR TITLE
allow removing multiple constraints at once

### PR DESCRIPTION
**Pull request by Irieo**
Make possible to pass a list into remove_constraints() to remove multiple constraints at once.
Keeps old functionality, i.e. it is still possible to pass constraint name as a string.
Adds logging, so user can see what constraints are removed.

to remove all constraints:
`n.model.remove_constraints(list(n.model.constraints))`

to remove a single constraint:
`n.model.remove_constraints(['Bus-nodal_balance'])`
or
`n.model.remove_constraints('Bus-nodal_balance')`

closes https://github.com/PyPSA/linopy/issues/210

### CODE REVIEW

**Review by FabianHofmann — APPROVED** [review]

**Comment by FabianHofmann** [comment]
Are you fine, if I set the logging from `info` to `debug`?

**Comment by Irieo** [comment]
> Are you fine, if I set the logging from `info` to `debug`?

sure!

File: linopy/model.py [hunk 1]
```diff
@@ -651,20 +651,27 @@ def remove_variables(self, name):
...
+            is provided, all constraints with those names will be removed.
 
         Returns
         -------
         None.
         """
-        self.constraints.remove(name)
+        if isinstance(name, list):
+            for n in name:
+                logger.info(f"Removed constraint: {n}")
```
Comment (FabianHofmann) [inline linopy/model.py:670]:
```suggestion
                logger.debug(f"Removed constraint: {n}")
```

File: linopy/model.py [hunk 2]
```diff
@@ -651,20 +651,27 @@ def remove_variables(self, name):
...
+                self.constraints.remove(n)
+        else:
+            logger.info(f"Removed constraint: {name}")
```
Comment (FabianHofmann) [inline linopy/model.py:673]:
```suggestion
            logger.debug(f"Removed constraint: {name}")
```

**Review by FabianHofmann — APPROVED** [review]

**Pull request merged**